.github/workflows/maven-ci.yml

```diff
@@ -27,10 +29,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
-    - name: Set up JDK 17
+    - name: Set up JDK 8
       uses: actions/setup-java@v4
       with:
-        java-version: '17'
+        java-version: '8'
```

Comment:
This will break HTTP and other projects requiring at least 17.

Reply:
you are right,
will try to fix all test that need jdk 8 in framework

https://github.com/apache/felix-dev/actions/runs/14275725289/job/40018259906#step:12:1151